packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -211,6 +215,8 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 							</div>
 						) }
 					</span>
+
+					{ showBundledBadge && <WooCommerceBundledBadge /> }
```

Comment:
Hi! In order not to make this part of code more complicated, can we make so that this badge is on the same logical layer as the existing `<PremiumBadge/>`? I'm thinking of moving this logic to the existing `getPricingDescription()`, something like this:

```
	function getPricingDescription() {

        ...

		let badge: React.ReactNode = null;
		if ( isPremium ) {
			badge = (
				<PremiumBadge
					tooltipPosition="bottom right"
					isPremiumThemeAvailable={ isPremiumThemeAvailable }
				/>
			);
		} else if ( showBundledBadge ) {
			badge = <WooCommerceBundledBadge />;
		}

		return (
			<div className="design-picker__pricing-description">
				{ badge }
				<span>{ text }</span>
			</div>
		);
	}
```

Another benefit is that we can still show the pricing text (`Free` / `Included in your plan`).

What do you think?

Reply:
In this case, the Woo badge would be positioned in the same place as the original Premium badge, right (not like the designs)?